launcher/build.gradle

```diff
@@ -2,59 +2,23 @@ buildscript {
     repositories {
         jcenter()
     }
-    dependencies {
-        classpath 'com.avast.gradle:gradle-docker-compose-plugin:0.9.4'
-    }
 }
 
 plugins {
     id 'org.springframework.boot'
-    id 'com.bmuschko.docker-spring-boot-application' version '5.0.0'
 }
 
-import com.bmuschko.gradle.docker.tasks.image.DockerTagImage
-
-apply plugin: 'docker-compose'
-
 springBoot {
     buildInfo()
 }
 
 bootBuildInfo.mustRunAfter processResources
 
-bootJar {
-    launchScript()
-    archiveFileName = 'web-app.jar' // will be placed in launcher/build/libs
-}
-
-docker {
-    springBootApplication {
-        tag = "REGISTRY.HOST.NAME/name/web-app:${version}"
-        baseImage = 'openjdk:11'
-        ports = [8080]
-        jvmArgs = ['-Dspring.profiles.active=prod', '-Xmx1024m']
-    }
-    registryCredentials {
-        url = 'https://REGISTRY.HOST.NAME/v2'
-        username = 'USERNAME'
-        password = 'container-repo-password'
-    }
-}
-
-task tagLatestImage(type: DockerTagImage) {
-    repository = "REGISTRY.HOST.NAME/name/web-app"
-    imageId = "REGISTRY.HOST.NAME/name/web-app:${version}"
-    tag = 'latest'
-}
-
-dockerBuildImage.finalizedBy tagLatestImage
-
-dockerCompose {
-    useComposeFiles = ['docker-compose.yml']
+bootBuildImage {
+    imageName = "your.organisation.here/$rootProject.name:$project.version"
```

Comment:
$project.group becomes the package name of the main class so it expanded to `engineering.everest.lhotse.launcher:lhotse:$version` which didn't seem that useful.

I couldn't find a more suitable Gradle project variable so went with 'your.organisation.here' so that it would be easy to search for.

I'm going to move the definition to the top most build file.